### ISSUE TITLE
404 when using aliases & cors

### ISSUE THREAD

**Issue by ColonelBundy** [ice-services/moleculer-web#30]
### Problem
Browser sends a ```OPTIONS``` request to determine cors and when using something like: 
```
aliases: {
  'POST login': 'v1.auth.login',
}
```
it won't work since ```OPTIONS``` request is not allowed. This will work, however:
```
aliases: {
  'OPTIONS login': 'v1.auth.login',
  'POST login': 'v1.auth.login',
}
```

**Comment by icebob** [comment]
Thanks. I need to reorder the processes in `httpHandler`. The aliases are processed sooner than CORS logic.

**Comment by icebob** [comment]
Released in 0.6.1
